src/detail-page/components/LearnTab.tsx

```diff
@@ -10,15 +11,21 @@ const TabContainer = styled('div')`
   width: 100%;
 `;
 
+const PROFESSION_CHARTS = {
+  frontend: null,
+  backend: null,
+  uxui: null,
+  qa: null,
+  devops: <DevOpsChart />,
+  projectmanager: null,
```

Comment:
I know it's not really part of your PR, but flat case is not really used in JS. Maybe we could change `uxui`, `devops` and `projectmanager` to camelCase? (`uxUi`, `devOps`, `projectManager`).